monitoring/regional_counts.py

```diff
@@ -2,23 +2,51 @@
 
 from utilities.db_helper import execute_sql_query
 
-if __name__ == "__main__":
+
+def get_case_receipts():
     sql_query = """
-SELECT CASE counts.region
-           WHEN 'E' THEN 'England'
-           WHEN 'W' THEN 'Wales'
-           WHEN 'N' THEN 'Northern Ireland'
-           ELSE 'Unknown Region: ' || counts.region END,
-       counts.region_count
-FROM (SELECT LEFT(region, 1) AS region, COUNT(*) AS region_count
-      FROM casev2.cases
-      WHERE receipt_received = true
-        AND action_plan_id = 'c4415287-0e37-447b-9c3d-1a011c9fa3db'
-      GROUP BY LEFT(region, 1)) AS counts
-    """
+    SELECT CASE counts.region
+               WHEN 'E' THEN 'England'
+               WHEN 'W' THEN 'Wales'
+               WHEN 'N' THEN 'Northern Ireland'
+               ELSE 'Unknown Region: ' || counts.region END,
+           counts.region_count
+    FROM (SELECT LEFT(region, 1) AS region, COUNT(*) AS region_count
+          FROM casev2.cases
+          WHERE receipt_received = true
+          AND action_plan_id = 'c4415287-0e37-447b-9c3d-1a011c9fa3db'
+          GROUP BY LEFT(region, 1)) AS counts
+        """
 
     db_result = execute_sql_query(sql_query)
 
     regional_counts = dict(db_result)
     regional_counts['Total'] = sum(regional_counts.values())
     print(json.dumps(regional_counts))
+
+
+def get_response_events():
+    sql_query = """
+    SELECT CASE counts.region
+               WHEN 'E' THEN 'England Responses'
+               WHEN 'W' THEN 'Wales Responses'
+               WHEN 'N' THEN 'Northern Ireland Responses'
+               ELSE 'Unknown Region: ' || counts.region END,
```

Comment:
I think the `Unkown Region` count would be indistinguishable from the receipted count above in the log metrics (not that we should actually see any unknown regions).